### ISSUE TITLE
Add CDN link

### ISSUE THREAD

**Issue by sbliven** [citation-js/replacer#7]
Can you add a link to a CDN version? I would like to try this out without building it myself.

I did find the package on jsdelivr (https://cdn.jsdelivr.net/npm/@citation-js/replacer@0.2.0/lib/index.min.js), but it doesn't include dependencies and I couldn't get it to work.

**Comment by larsgw** [comment]
This version of Replacer does not have a browser build, and so does not have a CDN link. This is because there are many possible combinations of versions and modules that could be included in the browser build. You can generate a bundle at the [bundle tool](https://juniper-coat.glitch.me/) ([source](https://github.com/citation-js/bundle-tool)) without having to setup a build environment. If you need different package requirements and you don't mind building it then the bundle tool can be a good starting point to setup a build environment anyway.

**Comment by sbliven** [comment]
The bundle tool is pretty good, I didn't know about it. However, it would still be useful to host the comprehensive bundle on a CDN so that it can be used with sites like codepen.io.

**Comment by larsgw** [comment]
Yeah, I'll have to think of something for that. I don't feel like having to copy the bundle environment to every package that needs it, but maybe I can figure out a way.
